docs/general/events.md

```diff
@@ -98,6 +98,15 @@ Recommendations on using attributes vs. body fields:
   * The _fields_ are unique to the named event (`event.name`) and different events
     may use the same _field_ name to represent different data, due to the unique
     nature of the event.
+  * The _fields_ SHOULD NOT use the `*.blob_ref.*` name, except when applying the
+    same principle/data model as "attribute references" but in relation to
+    event fields (e.g. using `somefield.blob_ref.uri` to relay the value of the
+    field `somefield` via indirection to an external storage system).
+  * For any _field_ name `somefield`, there exist a set of corresponding
```

Comment:
How would you represent when the entire body/payload is being referenced in blob storage? For example if the original payload was a large string/bytes/ArrayValue or you wanted to reference an entire KeyValueList and not subfields

Reply:
Thanks. My thought here is that a subfield named `blob_ref.uri` being present (no prefix before `blob_ref`) would indicate that the entire `AnyValue` (rather than a specific field of it) was written-aside.

I will amend this proposal to make that explicit...

Reply:
I've amended the PR to explicitly cover this case. OK to resolve?

Reply:
Yes thanks!